### PR TITLE
Use generic names for RedHat and Debian-like minions

### PR DESCRIPTION
**Pull request by Bischoff**
See uyuni-project/uyuni#5628

### CODE REVIEW

**Review by ktsamis — APPROVED** [review]

File: jenkins_pipelines/environments/manager-4.1-dev-acceptance-tests-NUE [hunk 1]
```diff
@@ -21,7 +21,7 @@ node('sumaform-cucumber') {
...
+            extendedChoice(name: 'functional_scopes',  multiSelectDelimiter: ',', quoteValue: false, saveJSONParameterToFile: false, type: 'PT_CHECKBOX', visibleItemCount: 30, value: '@scope_smdba,@scope_spacecmd,@scope_spacewalk_utils,@scope_visualization,@scope_notification_message,@scope_virtual_host_manager,@scope_subscription_matching,@scope_formulas,@scope_sp_migration,@scope_cve_audit,@scope_onboarding,@scope_content_lifecycle_management,@scope_res,@scope_recurring_actions,@scope_maintenance_windows,@scope_building_container_images,@scope_kubernetes_integration,@scope_openscap,@scope_dblike,@scope_action_chains,@scope_salt_ssh,@scope_tomcat,@scope_changing_software_channels,@scope_monitoring,@scope_salt,@scope_cobbler,@scope_sumatoolbox,@scope_virtualization,@scope_hub,@scope_retail,@scope_configuration_channels,@scope_content_staging,@scope_proxy,@scope_traditional_client,@scope_api,@scope_power_management,@scope_retracted_patches,@scope_ansible', description: 'Choose the functional scopes that you want to test')
```
Comment (ktsamis) [inline jenkins_pipelines/environments/manager-4.1-dev-acceptance-tests-NUE:24]: See where we use the scope rhlike_minion because I don't see it anywhere

**Review by ktsamis — CHANGES_REQUESTED** [review]

**Comment by vandabarata** [comment]
if I'm not mistaken, this PR is to add `scope_deblike` and `rhlike_minion`, right? I can't find the rhlike_minion :thinking:

**Review by srbarrios — APPROVED** [review]

**Comment by srbarrios** [comment]
> if I'm not mistaken, this PR is to add `scope_deblike` and `rhlike_minion`, right? I can't find the rhlike_minion thinking

We need a scope for debian like, because some features are specific of debian.
But I think we don't have anything similar for RH-like, that's why we don't have a scope_rhlike.

File: jenkins_pipelines/environments/manager-4.1-dev-acceptance-tests-NUE (hunk 1 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/manager-4.1-dev-acceptance-tests-NUE]: See also Oscar's answer to Vanda asking same question below

**Pull request merged**